Week2/homework/array.js

```diff
@@ -0,0 +1,24 @@
+//6.1 Declare an empty array. Make sure that the name you choose indicates 'plurality'.
+let animals = [];
+
+
+//6.2 Write a console.log statement that explains in words what you think the value of the array is.
+console.log('the content of my array will be about animals');
```

Comment:
Can you explain here what the value of the array is right now? (I know it will be about animals, but what is the value)

Reply:
It supposed to be an empty array, so I think the value here is 0!
I have changed this statement.